components/tag/index.css

```diff
@@ -209,54 +204,45 @@
 
 		margin-block-start: calc(var(--mod-tag-icon-spacing-block-start, var(--spectrum-tag-icon-spacing-block-start)) - var(--mod-tag-border-width, var(--spectrum-tag-border-width)));
 		margin-block-end: calc(var(--mod-tag-icon-spacing-block-end, var(--spectrum-tag-icon-spacing-block-end)) - var(--mod-tag-border-width, var(--spectrum-tag-border-width)));
-		margin-inline-end: var(--mod-tag-icon-spacing-inline-end, var(--spectrum-tag-icon-spacing-inline-end));
 		flex-shrink: 0;
 	}
 
 	.spectrum-Avatar {
 		margin-block-start: calc(var(--mod-tag-avatar-spacing-block-start, var(--spectrum-tag-avatar-spacing-block-start)) - var(--mod-tag-border-width, var(--spectrum-tag-border-width)));
 		margin-block-end: calc(var(--mod-tag-avatar-spacing-block-end, var(--spectrum-tag-avatar-spacing-block-end)) - var(--mod-tag-border-width, var(--spectrum-tag-border-width)));
-		margin-inline-end: var(--mod-tag-avatar-spacing-inline-end, var(--spectrum-tag-avatar-spacing-inline-end));
+	}
+
+	.spectrum-Tag-itemIcon,
+	.spectrum-Avatar,
+	.spectrum-Thumbnail {
+		/* set visual inline-start spacing, but take away label spacing, this will result in negative margin - this calc does not need border width */
+		margin-inline-start: calc(var(--mod-tag-visual-spacing-inline-start, var(--spectrum-tag-visual-spacing-inline-start)) - var(--mod-tag-label-spacing-inline, var(--spectrum-tag-label-spacing-inline)));
+		margin-inline-end: var(--mod-tag-visual-spacing-inline-end, var(--spectrum-tag-visual-spacing-inline-end));
 	}
 
 	/* clear button */
 	.spectrum-Tag-clearButton {
-		/* @passthrough start -- width of fill neutralized to ensure correct inline spacing within tag */
-		--mod-clear-button-width: fit-content;
-		--spectrum-clearbutton-fill-size: fit-content;
-		--spectrum-clearbutton-fill-background-color: transparent;
-		/* @passthrough end */
-
 		box-sizing: border-box;
 
-		padding-block-start: calc(var(--mod-tag-clear-button-spacing-block, var(--spectrum-tag-clear-button-spacing-block)) - var(--mod-tag-border-width, var(--spectrum-tag-border-width)));
-		padding-block-end: calc(var(--mod-tag-clear-button-spacing-block, var(--spectrum-tag-clear-button-spacing-block)) - var(--mod-tag-border-width, var(--spectrum-tag-border-width)));
+		margin-block-start: calc(var(--mod-tag-clear-button-spacing-block, var(--spectrum-tag-clear-button-spacing-block)) - var(--mod-tag-border-width, var(--spectrum-tag-border-width)));
+		margin-block-end: calc(var(--mod-tag-clear-button-spacing-block, var(--spectrum-tag-clear-button-spacing-block)) - var(--mod-tag-border-width, var(--spectrum-tag-border-width)));
```

Comment:
The padding for block spacing and margin for inline spacing that was here before was making the cross icon's focus outline distorted. It's definitely too tight as it is now. I'm kind of at a loss about what to do with this, open to ideas.

S1:
![image](https://github.com/user-attachments/assets/0b3ca6b5-e7fc-467e-bbd7-d10d0cdc61c0)

S2:
![image](https://github.com/user-attachments/assets/ebd3ebd9-c6f2-4820-a426-4ed71183b056)

Reply:
Leaving a todo comment for myself, since we talked about this: we'll remove the clear button focus outline/remove the clear button from the tabindex. And it would also be nice to make sure that the target for the clear button is of an accessible size.

Reply:
Resolving this as I think what I have now brings clear button to an accessible target size and matches the spacing in the spec.